### PR TITLE
New sound policies

### PR DESCRIPTION
**Pull request by DIOHz0r**
### Changes description

Added new detailed policies for devices' sounds

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### References

It updates the change made on PR #431

### CODE REVIEW

File: install/installer.class.php [hunk 1]
```diff
@@ -1377,14 +1377,146 @@ static public function getPolicies() {
...
+         ],
+
+         [
+            'name'                                => __('Disable ringer sounds', 'flyvemdm'),
+            'symbol'                              => 'disableStreamRing',
+            'group'                               => 'ui',
+            'type'                                => 'bool',
+            'type_data'                           => '',
+            'unicity'                             => 1,
+            'plugin_flyvemdm_policycategories_id' => 'Security > User interface',
```
Comment (btry) [inline install/installer.class.php]: Please, change category of these policies to "User interface" (without beinig a child of Security). We need to check the install and upgrade code handles correctly with 2 categories having the same short name.

File: install/upgrade/update_to_dev.php [hunk 2]
```diff
@@ -209,6 +209,11 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
       $migration->addPostQuery("DELETE FROM `$table` WHERE `symbol` IN ('" . $policiesStr . "')");
    }
 
+   // Add policies categories
+   $query = $query = "INSERT INTO `".PluginFlyvemdmPolicyCategory::getTable()."` VALUES (11,'Configuration',0,'Configuration',NULL,1,NULL,'[]'),
+      (12,'User interface',11,'Configuration > User interface',NULL,2,NULL,'{\"11\":\"11\"}')";
+   $DB->query($query) or plugin_flyvemdm_upgrade_error($migration);
+
```
Comment (btry) [inline install/upgrade/update_to_dev.php]: this is not needed. The code which creates the policies will generate the categories.

Comment (btry) [reply install/upgrade/update_to_dev.php]: FYI: https://github.com/flyve-mdm/glpi-plugin/blob/develop/install/installer.class.php#L329

File: install/installer.class.php [hunk 3]
```diff
@@ -1301,7 +1301,7 @@ static public function getPolicies() {
             'type'                                => 'bool',
             'type_data'                           => '',
             'unicity'                             => 1,
-            'plugin_flyvemdm_policycategories_id' => 'Security > User interface',
+            'plugin_flyvemdm_policycategories_id' => 'Security > Phone',
```
Comment (btry) [inline install/installer.class.php]: @DIOHz0r 
This policy is not related to phoning. I would rather keep it in "Security > User interface".

**Comment by DIOHz0r** [comment]
I've solved a merge conflict, please review it again.

**Comment by DIOHz0r** [comment]
@btry, could you review this policy again please?
